skrub/_tabular_learner.py

```diff
@@ -61,14 +61,14 @@ def tabular_learner(estimator, *, n_jobs=None):
 
```

Comment:
I think that we need to amend as well the documentation in the long summary just above where we already mentioned `"regressor"` and `"classifier"`.